src/components/form/form.types.ts

```diff
@@ -320,7 +320,128 @@ export enum FormLayoutType {
 }
 
 /**
- * @public
  * Represents the JSON schema with Lime specific options
+ * @public
+ */
+export interface FormSchema<T extends Record<string, any> = any>
```

Comment:
Is this correct or should `T` extend `JSONSchema7Definition` instead? :thinking:

Reply:
In this case T is not the Schema itself, but the object the schema represents.
So for example if you have the interface:

```typescript
interface Person {
    name: string;
    role: string;
}
```

then you can get a `FormSchema` based on the Person by `FormSchema<Person>`.
Then you get required and properties and such types with `keyof Person` and that kind of things